manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -22,7 +22,7 @@ metadata:
     containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
-    olm.skipRange: ">=4.1.0 <4.1.2"
+    olm.skipRange: ">=4.1.0 <4.1.0"
```

Comment:
same question as in https://github.com/openshift/cluster-logging-operator/pull/207/files#r299281610

Reply:
resolved in that discussion..  OLM won't choke on this and it's cleaner (imho) than the alternative.  If it becomes a problem for specific upgrade testing being done in this repo and we don't want the upgrade test to have to mutate this file, we can revisit at that point.